### DISCUSSION TITLE
ESP8684 ?

### DISCUSSION THREAD

**Discussion by Docteur-No** [micropython/micropython#16347] [General]
Hi all,
The ESP8684 is recent and an interesting successor to the ESP8266, when can we expect to have a micropython for this processor.
Thanks in advance for an answer.

Have a good day

**Comment by bidrohini** [comment]
Not sure about micropython. However, circuitpython is available: https://circuitpython.org/board/espressif_esp8684_devkitc_02_n4/

**Comment by mattytrentini** [comment]
The ESP8684 is a re-labelled ESP32-C2 - and there is a [fairly mature PR](https://github.com/micropython/micropython/pull/15906) for that micro.

**Comment by Docteur-No** [comment]
Many thanks for your quick answers !
Sorry for my ignorance but, what PR means ?

**Reply by mattytrentini** [reply]
PR = **P**ull **R**equest. It's represents a contribution to a project from someone - a request that changes are pulled into the repository. 

So, in this case, someone has added support for the ESP32-C2 and requested that it is integrated to the MicroPython project via a PR. That PR is being reviewed and tested and, when the core developers are satisfied, it can be merged.
